src/main/java/oak/utility/DateTimeUtility.java

```diff
@@ -0,0 +1,43 @@
+package oak.utility;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * The type Date time utility.
+ */
+public class DateTimeUtility {
+    /**
+     * Parse string to local datetime local date time.
+     *
+     * @param rawDateTime the raw datetime
+     * @return the local datetime
+     */
+    public static LocalDateTime parseStringToLocalDateTime(String rawDateTime) {
+        try {
+            return LocalDateTime.parse(rawDateTime);
+        } catch (DateTimeParseException e) {  }
+
+        // NOTE: We expect the user-input format to be: "<date> @ <time>"
+        String[] rawDateTimeArr = rawDateTime.split("@");
+        LocalDate date = LocalDate.parse(rawDateTimeArr[0].strip());;
+        LocalTime time = rawDateTimeArr.length > 1 ? LocalTime.parse(rawDateTimeArr[1].strip()) : LocalTime.MIDNIGHT;
```

Comment:
I guess it might be better to try to keep line length shorter than 110 characters. ^^